<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="LoginLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Enter login (email):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Enter password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Enter password again:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginEntery">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>Login (email)...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEntery">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEnteryAgain">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
